### PR TITLE
Player Inspection & Aero Theme Updates

### PR DESCRIPTION
**Pull request by Expertcoderz**
- Minor improvements and optimizations to the player inspection UI
- Some visual improvements to the Aero theme

_Screenshots of the Aero theme as requested:_
image
image
image
image
image
image
image
image

There are of course animations. I intend to update the theme regularly.

### CODE REVIEW

File: MainModule/Client/UI/Default/Inspect.lua [hunk 1]
```diff
@@ -122,12 +112,11 @@ return function(data)
 
 		addGeneralEntry("Username:", player.Name, "The player's Roblox username")
 		addGeneralEntry("User ID:", player.UserId, "The player's unique Roblox user ID")
-		addGeneralEntry("Account Age:", player.AccountAge .. " ("..string.format("%.2f", player.AccountAge/365).." years)", "How long (in days) the player has been registered on Roblox")
-		addGeneralEntry("Membership:", msTypeToStr(player.MembershipType), "The player's Roblox membership type")
+		addGeneralEntry("Account Age:", player.AccountAge .. " days ("..string.format("%.2f", player.AccountAge/365).." years)", "How long the player has been registered on Roblox")
+		addGeneralEntry("Membership:", tostring(player.MembershipType):sub(21), "The player's Roblox membership type")
```
Comment (fxeP1) [inline MainModule/Client/UI/Default/Inspect.lua:116]: You can use the `Name` property on Enum properties, so you should be able to simplify it only to `player.MembershipType.Name` instead.

**Review by fxeP1 — CHANGES_REQUESTED** [review]

File: MainModule/Client/UI/Default/Inspect.lua [hunk 2]
```diff
@@ -346,10 +335,31 @@ return function(data)
 
 		addAdonisEntry("Admin Level:", data.AdminLevel, "The player's Adonis rank")
 		addAdonisEntry("Donor:", boolToStr(data.IsDonor), "Is the player an Adonis Donor?")
-		addAdonisEntry("Contributor:", boolToStr(data.IsContributor), "Is the player an Adonis Contributor (Github)?")
 		addAdonisEntry("Muted:", boolToStr(data.IsMuted), "Is the player muted? (IGNORES TRELLO MUTELIST)")
 		addAdonisEntry("Banned:", boolToStr(data.IsBanned), "Is the player banned? (IGNORES TRELLO BANLIST)")
 
+		local extrainfo = nil
+		if player:GetRankInGroup(886423) == 10 then
+			extrainfo = "User is a GitHub contributor of Adonis"
```
Comment (P3tray) [inline MainModule/Client/UI/Default/Inspect.lua:343]: Change
`extrainfo = "User is a GitHub contributor of Adonis"`
To
`extrainfo = "User has contributed to Adonis via GitHub"`

File: MainModule/Client/UI/Default/Inspect.lua [hunk 3]
```diff
@@ -346,10 +335,31 @@ return function(data)
...
+		elseif player:GetRankInGroup(886423) == 12 then
+			extrainfo = "User is a developer of Adonis"
```
Comment (P3tray) [inline MainModule/Client/UI/Default/Inspect.lua:345]: Change
`extrainfo = "User is a developer of Adonis"`
To
`extrainfo = "User is an official developer of Adonis"`

File: MainModule/Client/UI/Default/Inspect.lua [hunk 4]
```diff
@@ -346,10 +335,31 @@ return function(data)
...
+		elseif player.UserId == 698712377 or player.UserId == 1237666 then
+			extrainfo = "You are inspecting the creator of Adonis"
```
Comment (P3tray) [inline MainModule/Client/UI/Default/Inspect.lua:347]: Change
`extrainfo = "You are inspecting the creator of Adonis"`
To
`extrainfo = "You are inspecting Sceleratis/Davey_Bones (Adonis creator)!`

**Review by P3tray — CHANGES_REQUESTED** [review]
Better sentences. You're welcome.

**Comment by LolloDev5123** [comment]
put screenshots in description of pull plzzzz

File: MainModule/Client/UI/Default/Inspect.lua (hunk 1 above)
Comment (Expertcoderz) [reply MainModule/Client/UI/Default/Inspect.lua:116]: Alright, thanks for the info.

**Comment by Sceleratis** [comment]
There seems to be a conflict

**Comment by P3tray** [comment]
Yeah. This conflicts with my removal of Groups from the datastore.

**Pull request merged**